File: nodejs/libs/redis.js
const Redis = require("ioredis");

const redisHosts = (process.env.FIZZBUZZ_REDIS_HOSTS || 'localhost').split(',');
console.log(redisHosts);
module.exports = redisHosts.length == 1 ? new Redis(6379, redisHosts[0]) : new Redis.Cluster(redisHosts.map(host => ({
  host,
  port: 6379
})));

File: rails/config/initializers/redis.rb
require "redis"

redis_hosts = (ENV["FIZZBUZZ_REDIS_HOSTS"] || "localhost").split(",")

Redis.current = redis_hosts.count == 1 ? Redis.new(host: redis_hosts.first) : Redis.new(cluster: redis_hosts.map { |h| { host: h, port: 6379 } })
